### ISSUE TITLE
Xen warning in xtest 1013

### ISSUE THREAD

**Issue by jforissier** [OP-TEE/optee_os#4715]
Platform: qemu_v8
```
$ make XEN_BOOT=y run
[...]
$ xtest 1013
[...]
o regression_1013.1.4 SHA-256 loop repeat 1000
(XEN) d0v0 Saved RPC cookie does not corresponds to OP-TEE's (ffff6439e9e00e00 != ffff6439e9e00a80)
(XEN) Xen WARN at optee.c:1147
(XEN) ----[ Xen-4.14.1  arm64  debug=n   Not tainted ]----
(XEN) CPU:    0
(XEN) PC:     000000000025c5ec optee.c#do_call_with_arg+0x4fc/0x510
(XEN) LR:     000000000025c5ec
(XEN) SP:     000080007ffdfd30
(XEN) CPSR:   80000249 MODE:64-bit EL2h (Hypervisor, handler)
(XEN)      X0: 00000000002f0028  X1: 0000000000000000  X2: 0000000000000000
(XEN)      X3: 0000000000000000  X4: 0000000000000005  X5: 0000000000000020
(XEN)      X6: 000080007fff8000  X7: 0000000000004000  X8: 00000000002fa000
(XEN)      X9: fefefefefefeff09 X10: 7f7f7f7f7f7f7f7f X11: 0101010101010101
(XEN)     X12: 0000000000000008 X13: 0000000000000011 X14: 000000000000000f
(XEN)     X15: 000000000000000f X16: 0000000000240680 X17: 0000000000000000
(XEN)     X18: 0000000000000000 X19: 000080007ffd3bc0 X20: ffff6439e9e00a80
(XEN)     X21: 000080007ffdfeb0 X22: 000080007dee7f60 X23: 00000000002f0280
(XEN)     X24: 000080007dee7f9c X25: 000000000029ba80 X26: 000000000028c810
(XEN)     X27: 6db6db6db6db6db7 X28: 0000000000000000  FP: ffff8000103abb60
(XEN)
(XEN)   VTCR_EL2: 80043594
(XEN)  VTTBR_EL2: 00010000b9f4b000
(XEN)
(XEN)  SCTLR_EL2: 30cd183d
(XEN)    HCR_EL2: 000000008078663f
(XEN)  TTBR0_EL2: 00000000bdeca000
(XEN)
(XEN)    ESR_EL2: f2000001
(XEN)  HPFAR_EL2: 0000000000080000
(XEN)    FAR_EL2: ffff800010010000
(XEN)
(XEN) Xen stack trace from sp=000080007ffdfd30:
(XEN)    000080007ffd3db0 00000000002f0280 0000000000000001 000080007ffdfeb0
(XEN)    000080007ffd3db0 00000000002f0280 000080007dee7f60 00000000002f0280
(XEN)    000080007dee7f9c 000000000029c000 000000000029b000 000000000029b000
(XEN)    000000000025d250 0000000000326008 00000000ffff0005 00000000ffff6439
(XEN)    00000000e9c22080 0000000000000000 0000000000000000 000080007dee71a8
(XEN)    000080007ffdfeb0 0000000032000003 000000005e000000 0000000060000005
(XEN)    000000005e000000 ffff6439f6f22438 00000000ffff0005 0000000000000000
(XEN)    ffff6439e9e84880 ffff6439e9ebd400 00000000002746dc ffff6439e25b7000
(XEN)    000000000029b000 000000000029b000 000080007def1000 0000000000000004
(XEN)    0000000000000004 000000000026e6c4 00000000002745c4 000080007def1e00
(XEN)    000080007ffdfeb0 000000005e000000 000080007ffdffb8 00000000002749d8
(XEN)    0000000000000000 ffff6439f6f22418 0000000000258458 000000000025844c
(XEN)    00000000ffff0005 00000000ffff6439 00000000e9c22080 0000000000000000
(XEN)    0000000000000000 0000000000000000 0000000000000000 0000000000000000
(XEN)    ffff8000103abbe8 ffff8000103ab9b0 00000000000009b0 000000000000000b
(XEN)    0000000000000000 000000000000025e 000000000000025e 0000783954750dda
(XEN)    0000000000000000 0000000000000000 0000000000000002 ffff6439f6f22400
(XEN)    ffff6439f6f22418 ffff8000103abc28 ffff8000103abbe8 ffff6439e9e84880
(XEN)    ffff6439f6f22438 00000000ffff0005 0000000000000000 ffff6439e9e84880
(XEN)    ffff6439e9ebd400 ffff8000103abb60 ffffabf9852d53d0 ffffffffffffffff
(XEN)    ffffabf984626600 5e00000060000005 0000000060000005 0000000000000000
(XEN)    0000000000000000 ffff6439e9ebd400 ffff8000103abb50 ffffabf984626604
(XEN)    0000000000000000 0000000000000000
(XEN) Xen call trace:
(XEN)    [<000000000025c5ec>] optee.c#do_call_with_arg+0x4fc/0x510 (PC)
(XEN)    [<000000000025c5ec>] optee.c#do_call_with_arg+0x4fc/0x510 (LR)
(XEN)
  regression_1013.1.4 OK
```

It is quite easy to reproduce. Anything to worry about?

Should "make check" detect these kind of messages and fail the test? (I'm thinking about https://github.com/OP-TEE/optee_os/pull/4706).

**Comment by lorc** [comment]
> Anything to worry about?

No, this is expected on that Xen version. This was fixed in 4.15:

http://xenbits.xen.org/gitweb/?p=xen.git;a=commit;h=aace5466996e743a087153ad3f9d8fcf11a76c56

**Comment by jforissier** [comment]
@lorc thank you. I will update the manifest.
